### ISSUE TITLE
Enable users to reference their own language as translation basis

### ISSUE THREAD

**Issue by ctgraham** [pkp/translator#51]
For users who are fluent in multiple languages, but who do not have English as a first language, it would be helpful to translate from a first language (e.g. using Spanish as a reference for Portuguese).

We could display the reference for the translation in the user's selected language, rather than in the `MASTER_LOCALE`.

**Comment by mtub** [comment]
Clinton, I think at some point we will have to give up the translator plugin - it still produces errors and we are thinking about a new translation workflow. That being said, I fear that your approach will not necessarily result in better translations. We cannot assess the quality of all translations, some of which are incomplete and far from up to date. Using those translations as a basis might result in chain reactions where errors will be multiplied from one locale to the other.

**Comment by ctgraham** [comment]
Yes, we recognize that the lifespan of this tool is probably short, in favor of a new workflow supporting industry standard tools and formats.  At the Pittsburgh Sprint, though, we heard of numerous missing translations, especially in user contributed plugins, where the local application managers have the interest in addressing the `##ubiquitous.octothorpes##`, even if just for their local context.  This may just be a feature branch which a select few want to merge in while the new workflow is still pending.
